platform-includes/performance/traces-sampler-as-sampler/java.spring.mdx

```diff
@@ -129,3 +158,40 @@ class CustomTracesSamplerCallback : TracesSamplerCallback {
   }
 }
 ```
+
+```kotlin {tabTitle:Kotlin (Spring Boot 4)}
```

Comment:
In this file all the Kotlin samples have `Kotlin (Spring Boot X)` instead of `Kotlin (Spring X)`